docs/_jekyll/.bundle/config

```diff
@@ -0,0 +1,3 @@
+---
+BUNDLE_PATH: "/Users/lolski/.bundle"
```

Comment:
These files should not be here.

Reply:
good find.

Reply:
additionally, i also removed a .ruby-version which i have carelessly added as well